craft_application/application.py

```diff
@@ -53,10 +53,11 @@
 class AppFeatures:
     """Specific features that can be enabled/disabled per-application."""
 
-    # Support for build-time secrets
     build_secrets: bool = False
-    # Support for add / modify package repositories
+    """Support for build-time secrets"""
+
     package_repository: bool = False
```

Comment:
what do you think about calling this `package_repositories`? And keep the plural everywhere, like the property in the model

Reply:
OK for me